exercises/a-safe-space-for-nulls/problem/problem.md

```diff
@@ -0,0 +1,77 @@
+Create a program that exports a `User` object instance to a CSV file using the null safe operator to access it's member properties.
+
+You will have a variable named `$user` available in your PHP script. We will put it there, each time your program runs  and it will be populated with random data each time it runs.
```

Comment:
>We will put it there, each time your program runs  and it will be populated with random data each time it runs.

probably drop the second mention of runs.. also additional whitespace

Reply:
Nice one!